frontend/public/components/modals/column-management-modal.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useDispatch } from 'react-redux';
+import * as _ from 'lodash';
+import {
+  Alert,
+  DataList,
+  DataListCheck,
+  DataListItem,
+  DataListItemRow,
+  DataListCell,
+  DataListItemCells,
+} from '@patternfly/react-core';
+
+import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory';
+import { setColumnManagementFilter } from '../../actions/ui';
+
+export const MAX_VIEW_COLS = 9;
+
+const OVERRIDES = ['Name'];
+
+const getDataListRows = (
+  rowData: any[],
+  onChange: (checked: boolean, event: any) => void,
+  disableUncheckedRows: boolean,
+): React.ReactNode[] =>
+  rowData.map((column) => (
+    <DataListItem aria-labelledby="table-column-management-item1" key={column.title}>
```

Comment:
I don't see an element with ID `table-column-management-item1`

Reply:
fixed to add dynamic labelled by strings based on unique ID